src/test-support/mocks/src/meshes/_.ts

```diff
@@ -1,7 +1,9 @@
 import type { EndpointDependencies, MockResponder } from '@/test-support'
 export default ({ fake, env }: EndpointDependencies): MockResponder => (req) => {
   const name = req.params.mesh
-  const mTLS = env('KUMA_MTLS_ENABLED', '')
+
+  const isMtlsEnabledOverride = env('KUMA_MTLS_ENABLED', '')
+  const isMtlsEnabled = isMtlsEnabledOverride !== '' ? isMtlsEnabledOverride === 'true' : fake.datatype.boolean()
```

Comment:
I think this means I can't use `0` or `false` in my bookmarklet/cookie anymore to make sure they are off? Is that right?

Reply:
Why not? Wouldn’t we get `isMtlsEnabledOverride === '0'`/`isMtlsEnabledOverride === 'false'` and so `isMtlsEnabled` would become `false`?

Reply:
Ah I see 🤔 , so what happens when KUMA_MTLS_ENABLED is `1`?

Reply:
It wouldn’t be enabled. For simplicity’s sake this should support the strings `'false'` and `'true'` with anything else defaulting for `false`.

Reply:
I always use `1` instead of writing `true` in my cookies as thats generally with env vars folks use both/either or.

Ok, let me change my bookmarklet for this one, I just have to change it the once for the MTLS one, and we are done. Feel free to resolve.